src/particles/feedback_CIC_gpu.cu

```diff
@@ -43,27 +48,81 @@ __device__ double atomicMax(double* address, double val)
 __global__ void initState_kernel(unsigned int seed, curandStateMRG32k3a_t* states) {
     int id = blockIdx.x*blockDim.x + threadIdx.x;
     curand_init(seed, id, 0, &states[id]);
-
 }
 
 
 /**
- * @brief Initialize the cuRAND state, which is analogous to the concept of generators in CPU code.
- * The state object maintains configuration and status the cuRAND context for each thread on the GPU.
- * Initialize more than the number of local particles since the latter will change through MPI transfers.
+ * @brief Does 2 things:
+ * -# Read in SN rate data from Starburst 99. If no file exists, assume a constant rate.
+ * -# Initialize the cuRAND state, which is analogous to the concept of generators in CPU code.
+ *    The state object maintains configuration and status the cuRAND context for each thread on the GPU.
+ *    Initialize more than the number of local particles since the latter will change through MPI transfers.
  *
- * @param n_local
+ * @param P pointer to parameters struct. Passes in starburst 99 filename and random number gen seed.
+ * @param n_local  number of local particles on the GPU
  * @param allocation_factor
  */
 void supernova::initState(struct parameters *P, part_int_t n_local, Real allocation_factor) {
```

Comment:
I'm glad we're starting to use more namespaces. It makes generic names like this more clear